cypress/integration/dashboard_spec.js

```diff
@@ -5,26 +5,25 @@ context('Dashboard View', () => {
 
   beforeEach(() => {
     cy.intercept('GET', coursesApi, { fixture: 'courses-api' })
-    cy.visit(baseUrl)
+    cy.visit(baseUrl).wait(300)
```

Comment:
Just curios why this was needed, i was under the impression cypress was built to not move forward until an item is found with the `find` command so i didn't think we needed this. would love to know more info!